ocaml/idl/datamodel.ml

```diff
@@ -5484,6 +5484,13 @@ let pif_set_property = call
     ~allowed_roles:_R_POOL_OP
     ()
 
+let pif_igmp_status =
+  Enum ("pif_igmp_status", [
+      "True", "The IGMP Snooping is enabled in the corresponding backend bridge.'";
+      "False", "The IGMP Snooping is disabled in the corresponding backend bridge.'";
+      "na", "The IGMP Snooping does not apply to this PIF or it is not initialized yet."
```

Comment:
How about "unknown"? For the description, we should explain the VLAN case: "The IGMP snooping status is unknown. If this is a VLAN master, then please consult the underlying VLAN slave PIF."